packages/neuron-wallet/src/keys/key.ts

```diff
@@ -0,0 +1,73 @@
+import bip32 from 'bip32'
+import bip39 from 'bip39'
+import { Keystore, Child } from './keystore'
+
+export default class Key {
+  private keystore: Keystore
+
+  private mnemonic: string
+
+  constructor(keystore: Keystore, mnemonic: string) {
+    this.keystore = keystore
+    this.mnemonic = mnemonic
+  }
+
+  public static fromKeystore = (keystore: Keystore) => {
+    return new Key(keystore, '')
+  }
+
+  public static fromKeystoreJson = (json: string) => {
+    return new Key(JSON.parse(json), '')
+  }
+
+  getKeystore = () => this.keystore
+
+  getKeystoreJson = () => JSON.stringify(this.keystore)
```

Comment:
why does the `getKeystoreJson` return a string

Reply:
Is json not a string?

Reply:
similar to an object

Reply:
JSON.parse ?

Reply:
this keystore in JSON.stringify is already a so called json, JSON.stringify transfer it to a string, and JSON.parse recover it to an object

Reply:
OK. I know and I will refactor later

Reply:
@Keith-CY If I want to provide json string format, what function name will be appropriate？

Reply:
`getKeystore` which accepts an `encode` parameter

Reply:
Like this ?
```
 getKeystore = (encode: boolean) => {
    return encode ? JSON.stringify(this.keystore) : this.keystore
  }
```